packages/transformer/src/IModelExporter.ts

```diff
@@ -82,6 +82,9 @@ export abstract class IModelExportHandler {
    */
   public shouldExportElement(_element: Element): boolean { return true; }
 
+  /** Called when element is skipped instead of exported. */
+  public onSkipElement(_element: Element): void { }
```

Comment:
can we make this new API take a parameter of `{ id: Id64String }` type? I didn't do a full check but on the surface the code that uses this looks like maybe it can take just an id. I'd like to eventually make it possible to not load the element if `shouldExportElement` returned false, and this API taking an element would violate that.

Reply:
I thought about only passing id at first, but saw that to get pending references to skipped element required all entity. I've changed onSkipElement to accept id directly, but had to also add `getReferencersByEntityKey` method to `PendingReferenceMap` class.